build.gradle

```diff
@@ -1,11 +1,14 @@
 plugins {
     id 'java'
+    id 'jacoco'
+    id 'com.github.kt3k.coveralls' version '2.8.4'
```

Comment:
Plz remove this deprecated coverall dependencies.(plugin) of course jaccoTestReport and coveralls on the buttom of the page